fapolicy_analyzer/ui/operations/deploy_changesets_op.py

```diff
@@ -168,6 +178,18 @@ def run(self, changesets: Sequence[Changeset], system: System, checkpoint: Syste
         -------
         None
         """
+
+        rules = (
+            bool(self.__modified_rules_text)
+            and self.__modified_rules_text != self.__rules_text
+        )
+        if rules:
+            unsaved_rule_dialog = UnsavedRulesDialog()
```

Comment:
Instead of rolling our own message dialog here we should use the built in `Gtk.MessageDialog` here. That would provide a more consistent feel to our message dialogs across the system. You can see an example of this here https://github.com/ctc-oss/fapolicy-analyzer/blob/fd5634b9df8a510afc56e6a3128173eda4815b44/fapolicy_analyzer/ui/add_file_button.py#L63